crates/wasmtime/Cargo.toml

```diff
@@ -63,6 +63,8 @@ hashbrown = { workspace = true, features = ["default-hasher"] }
 bitflags = { workspace = true }
 futures = { workspace = true, features = ["alloc"], optional = true }
 bytes = { workspace = true, optional = true }
+embedded-io = { version = "0.6.1", features = ["alloc"], optional = true }
```

Comment:
I definitely appreciate the utility of having `Read`/`Write` trait lookalikes here, but for purposes of keeping our TCB (and vetting overhead) small, and dependency base for other efforts (e.g. OOM handling) small as well, I think I'd prefer that we define our own `Read`/`Write` traits instead of pulling in this dependency.